test/integration/targets/eos_acls/tests/cli/_parsed_cfg.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Setup
+  cli_config:
```

Comment:
That also means these files have to be eos_config instead, which should arguably should make the tasks simpler

Reply:
Done